src/types.ts

```diff
@@ -0,0 +1,57 @@
+
+import React, { Dispatch } from "react"
+
+export type sectionData = {
```

Comment:
Types are typically capitalized like so

```suggestion
export type SectionData = {
```